src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesCipherTests.cs

```diff
@@ -442,6 +625,272 @@ public static void WrongKeyFailDecrypt_2()
+                key: "701ccc4c0e36e512ce077f5af6ccb957".HexToByteArray(),
+                iv: "5337ddeaf89a00dd4d58d860de968469".HexToByteArray(),
+                plainBytes: "cc1172f2f80866d0768b25f70fcf6361aab7c627c8488f97525d7d88949beeea".HexToByteArray(),
+                cipherBytes: "cdcf093bb7840df225683b58a479b00d5de5553a7e85eae4b70bf46dc729dd31".HexToByteArray(),
+                feedbackSize: 128);
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB128_192_NoPadding_0_Extended()
+        {
+            // NIST CAVP AESMMT.ZIP CFB128MMT192.rsp, [ENCRYPT] COUNT=0
+            TestAesTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "1bbb30016d3a908827693352ece9833415433618b1d97595".HexToByteArray(),
+                iv: "b2b48e8d60240bf2d9fa05cc2f90c161".HexToByteArray(),
+                plainBytes: "b4e499de51e646fad80030da9dc5e7e2".HexToByteArray(),
+                cipherBytes: "8b7ba98982063a55fca3492269bbe437".HexToByteArray(),
+                feedbackSize: 128);
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB128_192_NoPadding_1_Extended()
+        {
+            // NIST CAVP AESMMT.ZIP CFB128MMT192.rsp, [ENCRYPT] COUNT=1
+            TestAesTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "69f9d29885743826d7c5afc53637e6b1fa9512a10eea9ca9".HexToByteArray(),
+                iv: "3743793c7144a755768437f4ef5a33c8".HexToByteArray(),
+                plainBytes: "f84ebf42a758971c369949e288f775c9cf6a82ab51b286576b45652cd68c3ce6".HexToByteArray(),
+                cipherBytes: "a3bd28bb817bdb3f6492827f2aa3e6e134c254129d8f20dbc92389b7d89702d6".HexToByteArray(),
+                feedbackSize: 128);
+        }
+
+        [Theory]
+        [InlineData(CipherMode.CBC, 0)]
+        [InlineData(CipherMode.CFB, 128)]
+        [InlineData(CipherMode.CFB, 8)]
+        [InlineData(CipherMode.ECB, 0)]
+        public static void EncryptorReuse_LeadsToSameResults(CipherMode cipherMode, int feedbackSize)
+        {
+            // AppleCCCryptor does not allow calling Reset on CFB cipher.
+            // this test validates that the behavior is taken into consideration.
+            var input = "b72606c98d8e4fabf08839abf7a0ac61".HexToByteArray();
+
+            using (Aes aes = AesFactory.Create())
+            {
+                aes.Mode = cipherMode;
+
+                if (feedbackSize > 0)
+                {
+                    aes.FeedbackSize = feedbackSize;
+                }
+
+                using (ICryptoTransform transform = aes.CreateEncryptor())
+                {
+                    byte[] output1 = transform.TransformFinalBlock(input, 0, input.Length);
+                    byte[] output2 = transform.TransformFinalBlock(input, 0, input.Length);
+
+                    Assert.Equal(output1, output2);
+                }
+            }
```

Comment:
Let's go ahead and also create a decryptor and run one of the outputs through it twice, just to be sure.

Reply:
I'd also like to see this test repeated for the other symmetric algorithms (or have it pointed out to me that they already exist)